resolvconf.in

```diff
@@ -367,6 +367,9 @@ detect_init()
 			then
 				/etc/rc.d/$1 restart
 			fi'
+	elif test -d /etc/dinit.d && dinitctl=$(command -v dinitctl); then
+		# The only assertion we have is that dinitctl is in PATH
+		RESTARTCMD="'${dinitctl}' --quiet restart --ignore-stopped \$1"
```

Comment:
Please follow existing style. Something like this?

```
	elif [ -d /etc/dinit.d ] && type dinitctl >/dev/null 2>&1; then
		# The only assertion we have is that dinitctl is in PATH
		RESTARTCMD='dinitctl --quiet restart --ignore-stopped $1'
```

Reply:
POSIX:

`type` is [an extension](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/type.html). `command` is [not](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/command.html). Are you sure you want `type` over `command`?

Also, the `$()` syntax is [preferred](https://pubs.opengroup.org/onlinepubs/9699919799/xrat/V4_xcu_chap02.html#tag_23_02_06_03) over ` `` `:
> Because of these inconsistent behaviors, the backquoted variety of command substitution is not recommended for new applications that nest command substitutions or attempt to embed complex scripts.

Reply:
I *think* my rationale for preferring type over command has been long since fixed by the faulty upstream many years ago.
Although type is an extension, I don't know of any supported shell that doesn't actually have it.

I have no real preference now though, it can stay as command -v.

Reply:
Done.